src/components/main/SearchBar.js

```diff
@@ -0,0 +1,15 @@
+import "./SearchBar.css";
+import searchBarImg from "../../assets/Search.png";
+
+function SearchBar() {
+  return (
+    <div className="SearchBar">
+      <div>
+        <img src={searchBarImg} alt="search"></img>
```

Comment:
요런건 self closing tag 를 사용하면 좋을 것 같아요 :) 

```suggestion
        <img src={searchBarImg} alt="search"/>
```